Makefile

```diff
@@ -38,8 +43,10 @@ lint: ## Run golangci-lint
 verify: tidy generate format lint ## Verify the current code generation and lint
 	git diff --exit-code
 
-build-cli: ## Build the CLI binary
-	$(Q)go build -a -o ./bin/combo
+KUBERNETES_VERSION=v1alpha1
```

Comment:
I think we're interested in the Kubernetes version we're importing as dependencies, so in this case, [0.22.2](https://github.com/operator-framework/combo/blob/main/go.mod#L12). We can likely just hardcode this value for now, and bump this version when bumping Kubernetes versions in the root go.mod, and just take the extra maintenance burden over time. wdyt?

Reply:
What would you think about doing something like this so we can just automate grabbing that version?

```shell
go list -m all | grep k8s.io/apimachinery | awk '{print $2}'
```

We could just throw that into the `Makefile` and apply it as an `ldflag`.

Reply:
I think either hardcoding it or adding that shell command you pasted is fine, although it's a bit hacky. Maybe add a TODO comment to iron that out if we're interested in that approach?

Reply:
Yeah, it's a bit hacky, I agree. Let's just hardcode it for now and come back to it if it gets to be too annoying.